SimTKcommon/include/SimTKcommon/internal/ClonePtr.h

```diff
@@ -202,7 +207,11 @@ template <class T> class ClonePtr {
         other.reset(p);
         reset(otherp);
     }
-     
+
+    DEPRECATED_14("use get() instead")
+    const T* getPtr()  const  { return get(); }
+    DEPRECATED_14("use upd() instead")
+    T* updPtr() { return upd(); }
```

Comment:
In doxygen, there will be no indication that the method is deprecated. Instead, it will look like the method is undocumented but will be around indefinitely.

What about using the macro to **also** insert a doxygen comment for the method? In that case, even when C++14 rolls around, we will still want to use our macro. If you think that is desirable (to automatically insert doxygen comments to duplicate the compiler warning), then we could also remove the `_14` from the macro name.